comp/core/flare/flare.go

```diff
@@ -116,7 +116,22 @@ func (f *flare) onAgentTaskEvent(taskType rcclienttypes.TaskType, task rcclientt
 		return true, fmt.Errorf("User handle was not provided in the flare agent task")
 	}
 
-	filePath, err := f.Create(nil, 0, nil)
+	enableProfiling, found := task.Config.TaskArgs["enable_profiling"]
+	if !found {
+		f.log.Debug("enable_profiling arg not found, creating flare without profiling enabled")
+		enableProfiling = "false"
+	}
+
+	flareArgs := types.FlareArgs{}
+	if enableProfiling == "true" {
+		// RC expects the agent task operation to provide reasonable default flare args
+		flareArgs.ProfileDuration = time.Second * 60
+		// flareArgs.ProfileMutex = ...
+	} else if enableProfiling != "false" {
+		f.log.Warnf("Unrecognized value passed via enable_profiling, creating flare without profiling enabled: %s", enableProfiling)
+	}
```

Comment:
Should this be a warning ? Customers have no way to mitigate this error and it might trigger monitor on their side. An info would be better I think.

Reply:
Sold